### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.5.2 - provide a mechanism to hide carousel controls on small touch deveices

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1307]
# Feature Request

## Description
Currently, touch devices show [carousel ](https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/carousel)controls on down-event (hold). This can not be reversed by the user, even though the controls obsure parts of the slide. 

## Solution
The controls could disapper on tap outside of the carousel in order to meet the success criterion. However, it would be better to always show the controls on either side of the carousel, not on top of the slides. 

## Alternative
The carousel has a number of other issues, such as poor colour contrast, a flaw in html semantics, a missing aria-role and no stop/pause button. Recommendation: rebuild carousel according to W3C tutorial https://www.w3.org/WAI/tutorials/carousels/)

**Comment by benjaminkott** [comment]
Controls are now always shown for all devices

**Comment by SarahEdding** [comment]
QA done. alle fixed
